app/config/security.yml

```diff
@@ -1,4 +1,7 @@
 security:
+    # Providers auth exceptions must not be hidden to customise the error message
+    # see usages of App\Security\Exception\BadCredentialsException
+    hide_user_not_found: false
```

Comment:
We need to handle it for the main admin too then, only the main is covered here by modifying the user provider, as far as I can see.

Reply:
It's ok because admin and user provider used the same class :

```
<service id="app.security.admin_provider" class="AppBundle\Security\UserProvider">
```

Reply:
Ok I've forgot that, thx!